manifests/resource/location.pp

```diff
@@ -7,8 +7,7 @@
 #     (present|absent)
 #   [*internal*]             - Indicates whether or not this location can be
 #     used for internal requests only. Default: false
-#   [*server*]                - Defines the default server for this location
-#     entry to include with
+#   [*server*]                - Defines all server for this location entry to include with
```

Comment:
This documentation doesn't read very well.  Could you try rephrasing?

Reply:
What about 'Defines a server or list of servers that integrate these location'?

Reply:
I changed the text, hopefully it's better than the last try :)